pyface/ui/qt4/console/console_widget.py

```diff
@@ -1511,7 +1516,12 @@ def _format_as_columns(self, items, separator="  "):
 
         # Calculate the number of characters available.
         width = self._control.viewport().width()
-        char_width = QtGui.QFontMetrics(self.font).width(" ")
+        # QFontMetrics.width() is deprecated and Qt docs suggest using
```

Comment:
Would it make sense to require a minimum version of Qt 5.12 (say)? (My fear is that that second branch will never actually be exercised in practice.) Do we know of any projects that are still using older versions of Qt?